src/main/java/com/snowflake/kafka/connector/internal/SnowflakeSinkServiceV1.java

```diff
@@ -316,14 +315,13 @@ private ServiceContext(String tableName, String stageName,
       this.buffer = new PartitionBuffer();
       this.ingestionService = conn.buildIngestService(stageName, pipeName);
       this.prefix = FileNameUtils.filePrefix(conn.getConnectorName(), tableName, partition);
-      this.processedOffset = new AtomicLong(-1);
```

Comment:
I am wondering if it is better to leave this stuff as is and pass these in to pipestatus when you need to ? THoughts @sfc-gh-bli